src/Traits/Debouncable.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Mpbarlow\LaravelQueueDebouncer\Traits;
+
+use Mpbarlow\LaravelQueueDebouncer\Facade\Debouncer;
+
+trait Debouncable
```

Comment:
Would you mind renaming this trait to Debounc**e**able please? I'm thinking it's sort of the opposite of "bounceable".